hera_sim/visibilities/conversions.py

```diff
@@ -53,8 +53,8 @@ def uvbeam_to_lm(uvbeam, freqs, n_pix_lm=63, trunc_at_horizon=False, **kwargs):
     Returns:
         ndarray, shape[nfreq, beam_px, beam_px]: the beam map cube.
     """
-
-    l = np.linspace(-1, 1, n_pix_lm, dtype=np.float32)
+    
+    l = np.linspace(-1, 1, n_pix_lm, dtype=np.float64)
```

Comment:
are you sure about this? Since `np.float32` is not default, I'm sure I had a good reason to set it explicitly to that.

Reply:
Not sure, so I switched it back to `np.float32`.